code/game/objects/items/devices/taperecorder.dm

```diff
@@ -138,12 +157,10 @@
 		return
 
 	if(mytape.used_capacity < mytape.max_capacity)
-		to_chat(usr, "<span class='notice'>Recording started.</span>")
 		playsound(src, 'sound/items/taperecorder/taperecorder_play.ogg', 50, FALSE)
-		recording = 1
+		recording = TRUE
+		say("<font color='[say_color]'>Recording started.</font>")
```

Comment:
How drastic is the difference between this and \<span class='notice'\>?

Reply:
Well it's maroon so pretty drastic. I think I could figure out how to make it use spans though.